ui/src/store/index.ts

```diff
@@ -11,7 +11,7 @@ import { support, SupportState } from "./modules/support";
 import { spinner, SpinnerState } from "./modules/spinner";
 import { snackbar, SnackbarState } from "./modules/snackbar";
 import { sessions, SessionsState } from "./modules/sessions";
-import { security, SecurityState } from "./modules/security";
+import { sessionRecording, SessionRecordingState } from "./modules/security";
```

Comment:
It wouldn't make sense to rename the model as well, as the model name is now misleading.

Reply:
Ignore this comment because I found that you did the change.